uart_com.cpp

```diff
@@ -452,3 +476,12 @@ void uart_com::check_feedback()
   }
 }
 
+void uart_com::check_CAP12xx_logging()
+{
+  String CAP12xx_logmsg ="CAP12xx = " + String(this->general) + "," + String(this->noise) +"," + String(this->cal_act) + "," + String(this->i2c_err) + "," + String(this->uart_err);
+  if(send_count >= 1000){
```

Comment:
Why don't you use a flag instead of a count? It will output as the handle micro controller emits.